src/main/java/Chat.java

```diff
@@ -0,0 +1,97 @@
+import java.util.*;
```

Comment:
maybe the import statement could explicitly mention that ArrayList class and scanner class are used.